eds/blocks/storyteller/storyteller.js

```diff
@@ -160,7 +168,24 @@ export default async function decorate(block) {
     }
   }
 
-  if ((pictureTagLeft === null) && (vidUrls.length >= 1)) {
+  if ((pictureTagLeft === null) && (vidUrls.length === 0)) {
+    const h2Tags = block.querySelectorAll('h2');
+    if (h2Tags.length > 1) {
+      setforegroundContainersNoVideo(block);
+      const foregroundWrapper = block.querySelector('.foreground-container');
+      foregroundContentContainer.classList.add('foreground-content');
+      foregroundContent.appendChild(h2Tags[1]);
+      if (pTags.length > 5) {
+        foregroundContent.appendChild(pTags[pTags.length - 1]);
+      } else {
+        foregroundContent.appendChild(pTags[4]);
```

Comment:
Won't this only work when pTags.length === 5? (And in that case, isn't 4 === pTags.length - 1 anyway?)

Reply:
Simplified, no need for if.